installer/tue-install-impl.bash

```diff
@@ -744,13 +734,14 @@ fi
 for target in $targets
 do
     tue-install-debug "Main loop: installing $target"
-    tue-install-target $target
+    tue-install-target $target || tue-install-error "Installed target: '$target' doesn't exist (anymore)"
 
-    if [[ -d $TUE_INSTALL_TARGETS_DIR/$target && "$tue_cmd" == "install" ]]
-    then
-        # Mark as installed
+    if [[ "$tue_cmd" == "install" ]]
```

Comment:
Did you remove the check for target directory because it is already being checked inside the function tue-install-target ?

Reply:
Indeed.